### ISSUE TITLE
Use this with Tally Arbiter

### ISSUE THREAD

**Issue by SamVimes78** [aaronpk/atem-tally-controller#10]
I suspect these tally lights are the reason why my ATEM Mini Pro freezes during a live stream. It still outputs the currently selected camera through USB to my streaming PC but i can't switch anything on the ATEM or even remotely.

Someone in the Blackmagic forums said this behaviour occurs when too many devices try to communicate with the ATEM over the network. If a network device looses its connection to the ATEM and reconnects, the new connection doesn't replace the old one but is registered as an additional connection.

My theory:
The tally lights frequently lose their wifi connection and reconnect to the ATEM, causing a freeze.

The ATEM always "recovers" after a few minutes (less than 5) but after that i'm stuck without remote controls and can only use the hardware switches on the ATEM.

My proposed solution:
Don't let the tally lights communicate directly with the ATEM but use Tally Arbiter instead. I'm using the Bitfocus Compantion software to control the ATEM and the OBS software over a Stream Deck macro keyboard. Bitfocus Companion can communicate with Tally Arbiter and that can control the Tallys.

There already seems to be a code to use the M5StickC with Tally Arbiter. But your code has the ability to select the camera and rotate the tally.

**Comment by aaronpk** [comment]
Yes, I would recommend running this through Tally Arbiter instead for the reasons you mentioned. I don't have plans to update this at the moment, but it shouldn't take much work to make this communicate with Tally Arbiter instead.

**Comment by SamVimes78** [comment]
Did you run into the same issue?
